src/components/Components/Widgets/RecordStatusWidget.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import {
+  colorBadgeBlue,
+  colorBadgeGreen,
+  colorBadgeRed,
+  colorPrimary
+} from '../../../styles/styleVariables';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+// Children
+import DropdownMenu from './DropdownMenu';
+import ClickableOverlay from '../ClickableOverlay';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+
+// TODO: Remove this default value vvvv
+const RecordStatusWidget = ({ value = 'open_record', onChange }) => {
+  const [open, setOpen] = useState(false);
+
+  const handleSelectOption = option => {
+    setOpen(false);
+    if (option !== value) onChange(option);
+  };
+
+  const buildActions = () => {
+    return {
+      normal: [
+        {
+          display: 'Open Record',
+          onClick: () => handleSelectOption('open_record')
+        },
+        {
+          display: 'Redacted',
+          onClick: () => handleSelectOption('redacted')
+        },
+        {
+          display: 'Restricted',
+          onClick: () => handleSelectOption('restricted')
+        }
+      ],
+      caution: [
+        {
+          display: 'Non-record',
+          onClick: () => handleSelectOption('non-record')
+        }
+      ]
+    };
+  };
+
+  const mapValueToDisplayValue = () => {
+    const mapping = {
+      open_record: 'Open record',
+      'non-record': 'Non-record',
```

Comment:
Why does this have single quotes?

Reply:
it JS, keys can be of any type, or no type. Sometimes you need to make a key a string to fit js syntax-- the hyphen in "non-record" breaks without quotes.